ibex_core.core

```diff
@@ -25,6 +25,7 @@ filesets:
       - rtl/ibex_multdiv_slow.sv
       - rtl/ibex_prefetch_buffer.sv
       - rtl/ibex_pmp.sv
+      - rtl/ibex_writeback.sv
```

Comment:
Shall we rename this file and the module to `ibex_wb_stage.sv` to match the current style?

Reply:
Done

Reply:
:+1: